doc/architecture-decisions/0017-moving-istio-configuration-out-of-this-repo.md

```diff
@@ -0,0 +1,44 @@
+# 17. Moving Istio and Related Configuration to CF-for-K8s Repo
+
+Date: 2020-09-15
+
+## Status
+
+Accepted
+
+## Context
+
+This ADR reverts the decision made in [ADR # 7. Maintain Generated
+Istio](./0007-maintain-generated-istio.md).
+
+The networking config and related Istio config is spread widely throughout both
+cf-for-k8s and cf-k8s-networking. Having the config in both places has made
+processes such as updating networking config, versioning routecontroller,
+upgrading Istio, deciding where some networking config should exist (in this repo or
+in the cf-for-k8s repo), and so on complicated.
+
+
+## Decision
+
+Relint and Networking came to a consensus to do move Istio configuration to
```

Comment:
the word `Relint` might not well known in the community :) maybe we can use a more standard name for this.